config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql

```diff
@@ -0,0 +1,59 @@
+  PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+  PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+  PREFIX dcterms: <http://purl.org/dc/terms/>
+  PREFIX astreams: <http://www.w3.org/ns/activitystreams#>
+  PREFIX org: <http://www.w3.org/ns/org#>
+
+  DELETE {
+    GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris ?p ?o.
+    }
+  }
+  INSERT {
+    GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris a astreams:Tombstone.
+      ?mandataris dcterms:modified ?now .
+      ?mandataris astreams:deleted ?now .
+      ?mandataris astreams:formerType mandaat:Mandataris.
+
+      ?membership a astreams:Tombstone.
+      ?membership dcterms:modified ?now .
+      ?membership astreams:deleted ?now .
+      ?membership astreams:formerType org:Membership .
+    }
+  }
+ WHERE {
+    VALUES (?mandataris ?persoon) {
```

Comment:
remove person from query, it doesn't serve a purpose

Reply:
Removed